cassandane/utils/tiny-test-splitter.pl

```diff
@@ -56,7 +56,6 @@
 
 LINE: while ($_ = shift @lines) {
   if (/^sub test_(\S+)/) {
-    my $file = $1 =~ s/_/-/gr;
```

Comment:
Does this change work?  Looks to me like `$file` is no longer declared

Reply:
Woof, thanks.  Of course this file isn't covered by tests.